src/Stratis.Bitcoin.Features.SmartContracts/Wallet/SmartContractWalletFeature.cs

```diff
@@ -1,139 +1,66 @@
-﻿using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Logging;
-using NBitcoin;
+using NBitcoin.Policy;
 using Stratis.Bitcoin.Builder;
 using Stratis.Bitcoin.Builder.Feature;
 using Stratis.Bitcoin.Configuration.Logging;
-using Stratis.Bitcoin.Connection;
 using Stratis.Bitcoin.Features.Wallet;
-using Stratis.Bitcoin.Features.Wallet.Broadcasting;
 using Stratis.Bitcoin.Features.Wallet.Interfaces;
-using Stratis.Bitcoin.Features.Wallet.Services;
-using Stratis.Bitcoin.Utilities;
 
 namespace Stratis.Bitcoin.Features.SmartContracts.Wallet
 {
     public sealed class SmartContractWalletFeature : FullNodeFeature
     {
-        private readonly BroadcasterBehavior broadcasterBehavior;
-        private readonly ChainIndexer chainIndexer;
-        private readonly IConnectionManager connectionManager;
         private readonly ILogger logger;
-        private readonly IWalletManager walletManager;
-        private readonly IWalletSyncManager walletSyncManager;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="WalletFeature"/> class.
         /// </summary>
-        /// <param name="broadcasterBehavior">The broadcaster behavior.</param>
-        /// <param name="chainIndexer">The chain of blocks.</param>
-        /// <param name="connectionManager">The connection manager.</param>
-        /// <param name="walletManager">The wallet manager.</param>
-        /// <param name="walletSyncManager">The synchronization manager for the wallet, tasked with keeping the wallet synced with the network.</param>
-        public SmartContractWalletFeature(
-            BroadcasterBehavior broadcasterBehavior,
-            ChainIndexer chainIndexer,
-            IConnectionManager connectionManager,
-            ILoggerFactory loggerFactory,
-            IWalletManager walletManager,
-            IWalletSyncManager walletSyncManager,
-            INodeStats nodeStats)
+        public SmartContractWalletFeature(ILoggerFactory loggerFactory)
```

Comment:
You might as well remove the loggerfactory here as well and use NLog's LogManager.GetCurrentClassLogger below ;)

Reply:
Done.